jp/components/snackbar.md

```diff
@@ -194,44 +194,13 @@ public restore() {
 
 <div class="divider--half"></div>
 
-### API まとめ
+### API リファレンス
 
-以下の表は便利な **igx-snackbar** コンポーネントの入力、出力、およびメソッドを説明します。
+このトピックでは、[`IgxSnackbarComponent`]({environment:angularApiUrl}/classes/igxsnackbarcomponent.html) を使用と構成方法を説明しました。API の詳細については以下のリンク先を参照してください。
 
-#### 入力
+* [`IgxSnackbarComponent`]({environment:angularApiUrl}/classes/igxsnackbarcomponent.html)
```

Comment:
Provide Sassdoc reference for the [igx-snackbar-theme](https://staging.infragistics.local/products/ignite-ui-angular/docs/sass/#function-igx-snackbar-theme)